data/json/recipes/recipe_weapon.json

```diff
@@ -2221,6 +2221,873 @@
+    [
+      [ "oxy_torch", 3 ],
+      [ "welder", 15 ],
+      [ "welder_crude", 25 ],
+      [ "toolset", 15 ],
+      [ "small_repairkit", 15 ],
+      [ "large_repairkit", 15 ]
+    ]
+  ],
+  "components": [
+    [
+      [ "scrap", 6 ],
+      [ "pipe", 1 ],
+      [ "steel_chunk", 2 ]
+    ],
+    [
+      [ "plastic_chunk", 3 ]
+    ],
+    [
+      [ "spring", 2 ]
+    ]
+  ]
+},{
+  "type" : "recipe",
+  "result": "red_dot_sight",
+  "category": "CC_WEAPON",
+  "subcategory": "CSC_WEAPON_MODS",
+  "skill_used": "electronics",
+  "difficulty": 6,
+  "time": 20000,
+  "reversible": false,
+  "autolearn": false,
+  "book_learn": [[ "manual_electronics", 3 ] , [ "textbook_electronics", 3 ]],
+  "qualities" : [
+    {"id":"SCREW_FINE","level":1,"amount":1}
+  ], "tools": [
+    [
+      [ "soldering_iron", 10 ],
+      [ "toolset", 10 ]
+    ]
+  ],
+  "components": [
+    [
+      [ "amplifier", 1 ]
+    ],
+    [
+      [ "power_supply", 1 ]
+    ],
+    [
+      [ "lens", 3 ]
+    ],
+    [
+      [ "battery", 100 ],
+      [ "plut_cell", 1 ]
+    ],
+    [
+      [ "scrap", 4 ],
+      [ "can_drink", 1 ],
+      [ "can_food", 1 ],
+      [ "canister_empty", 1 ],
+      [ "plastic_chunk", 1 ]
+    ],
+    [
+      [ "cable", 5 ]
+    ]
+  ]
+},{
+  "type" : "recipe",
+  "result": "rifle_scope",
+  "category": "CC_WEAPON",
+  "subcategory": "CSC_WEAPON_MODS",
+  "skill_used": "fabrication",
+  "difficulty": 10,
+  "time": 20000,
+  "reversible": false,
+  "autolearn": true,
+  "book_learn": [[ "textbook_fabrication", 5 ] , [ "manual_fabrication", 5] , [ "manual_gun", 6]],
+  "qualities":[
+    {"id":"HAMMER_FINE","level":1,"amount":1},
+    {"id":"SAW_M_FINE","level":1,"amount":1}
+  ],
+  "tools": [
+    [
+      [ "soldering_iron", 10 ],
+      [ "toolset", 10 ],
+      [ "small_repairkit", 10 ],
+      [ "large_repairkit", 10 ]
+    ]
+  ],
+  "components": [
+    [
+      [ "lens", 2 ]
+    ],
+    [
+      [ "can_drink", 1 ],
+      [ "can_food", 1 ],
+      [ "canister_empty", 1 ]
+    ]
+  ]
+},{
+  "type" : "recipe",
+  "result": "spare_mag",
+  "category": "CC_WEAPON",
+  "subcategory": "CSC_WEAPON_MODS",
+  "skill_used": "fabrication",
+  "skills_required": [[ "gun", 2 ]],
+  "difficulty": 4,
+  "time": 15000,
+  "reversible": false,
+  "autolearn": true,
+  "book_learn": [[ "manual_fabrication", 2 ] , [ "manual_pistol", 3] , [ "manual_shotgun", 4] , [ "manual_smg", 3] , [ "manual_rifle", 3]],
+  "qualities":[
+    {"id":"HAMMER_FINE","level":1,"amount":1},
+    {"id":"SCREW_FINE","level":1,"amount":1}
+  ],
+  "tools": [
+    [
+      [ "large_repairkit", 20 ],
+      [ "small_repairkit", 60 ]
+    ]
+  ],
+  "components": [
+    [
+      [ "scrap", 3 ],
+      [ "steel_chunk", 1 ]
+    ],
+    [
+      [ "spring", 1 ]
+    ]
+  ]
+},{
+  "type" : "recipe",
+  "result": "stabilizer",
+  "category": "CC_WEAPON",
+  "subcategory": "CSC_WEAPON_MODS",
+  "skill_used": "mechanics",
+  "skills_required": [[ "electronics", 5 ]],
```

Comment:
I'm perfectly content to just block the gyro-stabilizer altogether (shop/clean-room territory) but at minimum it ought to require a good working knowledge of guns to understand how it'd counter the recoil.  Figure adding gun 5-6 or so in the required-skills list.